packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/components/WorkflowEditActionFormServerlessFunction.tsx

```diff
@@ -77,8 +77,6 @@ export const WorkflowEditActionFormServerlessFunction = ({
   const theme = useTheme();
   const { getIcon } = useIcons();
   const serverlessFunctionId = action.settings.input.serverlessFunctionId;
-  const serverlessFunctionVersion =
```

Comment:
Are we sure we don't rely on `action.settings.input.serverlessFunctionVersion` elsewhere instead of `draft` as you did here?